packages/template-block/src/helpers/layout.ts

```diff
@@ -20,14 +20,10 @@ export const getCardPadding = (blockSettings: Settings) => {
 
 export const getIsRows = (hasPreview: boolean, textPositioning: TextPositioningType): boolean => {
     const flexDirection = hasPreview ? textPositioningToFlexDirection[textPositioning] : 'row';
-
-    return hasPreview && (flexDirection === 'row' || flexDirection === 'row-reverse');
+    return hasPreview && ['row', 'row-reverse'].includes(flexDirection);
 };
```

Comment:
```typescript
export const getIsRows = (hasPreview: boolean, textPositioning: TextPositioningType): boolean => {
    return hasPreview && ['row', 'row-reverse'].includes(textPositioningToFlexDirection[textPositioning]);
};
``` 

How about that?

Reply:
Well spotted, this is a very weird double-condition for some reason.